packages/algolia/algolia-schema.json

```diff
@@ -46,7 +46,7 @@
   ],
   "attributesToSnippet": null,
   "attributesToHighlight": null,
-  "paginationLimitedTo": 1000,
+  "paginationLimitedTo": 3000,
```

Comment:
on the export function the page size limit is set to 10000, I think that it will cause an error once again when the number of RO increases, this is set in this variable `MAX_ALGOLIA_RESULTS`

Reply:
https://github.com/yldio/asap-hub/blob/master/apps/crn-frontend/src/shared-research/export.ts#L10